src/network/networkd-network.h

```diff
@@ -275,6 +277,8 @@ struct Network {
         char **ntp;
         char **sip;
         char **bind_carrier;
+
+        OrderedHashmap *traffic_control_queue_discs;
```

Comment:
Why don't you use `LIST` instead of `OrderedHashmap`?

Reply:
I think this a much better way to handle the ordered way .